### ISSUE TITLE
Separate endpoint for creating contests

### ISSUE THREAD

**Issue by MorganLove** [votingworks/arlo#326]
There should be a separate endpoint for creating contests, instead of using the `/basic` endpoint.

**Comment by mcchilders** [comment]
This is a placeholder card requested by @benadida to hold space for future work (see https://github.com/votingworks/arlo/pull/318#pullrequestreview-370800440 for the initial convo that prompted this.) Don't start this until we have a conversation!

**Comment by nikhilb4a** [comment]
Completed back in 2020
